src/index.html

```diff
@@ -3,12 +3,64 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <div class="menu__content">
+        <a href="#home">
+          <img class="logo" src="./images/logo.png"
+          alt="logo">
```

Comment:
please fix the code styling according to the checklist. fix all places in HTML file
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/cddaa98d-2d68-445b-9c7d-1183e232b176)